osbs/build/build_request.py

```diff
@@ -235,6 +241,7 @@ def render_orchestrate_build(self):
             'source_registry_uri': self.spec.source_registry_uri.value,
             'sources_command': self.spec.sources_command.value,
             'vendor': self.spec.vendor.value,
+            'equal_labels': equal_labels_string,
```

Comment:
Let's set this to `None` if falsy because a couple of lines below, we remove keys that have `None` values.